lib/ansible/template/__init__.py

```diff
@@ -68,6 +67,17 @@
 
 JINJA2_OVERRIDE = '#jinja2:'
 
+USE_JINJA2_NATIVE = False
+if C.DEFAULT_JINJA2_NATIVE:
+    try:
+        from jinja2.nativetypes import NativeEnvironment as Environment
+        from jinja2.nativetypes import native_concat as j2_concat
+        USE_JINJA2_NATIVE = True
+    except ImportError:
+        from jinja2 import Environment
```

Comment:
What about doing the same with `from jinja2.utils import concat as j2_concat`?

Reply:
Done, thanks

Reply:
Cool. I think you can drop line 41 too.